.github/workflows/tests_branch.yml

```diff
@@ -46,7 +46,7 @@ jobs:
 
       - name: Run python tests
         run: |
-          python3 -m pytest
+          python3 -m pytest --verbose --verbose --cov-config=pyproject.toml --cov .
```

Comment:
I'm getting you on little stuff today but here we go: double `--verbose`

Reply:
Oh wow! That's great actually, want a more verbose mode? Just throw another verbose in there